RadixWallet/Core/DesignSystem/Components/AppTextField.swift

```diff
@@ -99,30 +99,32 @@ public struct AppTextField<FocusValue: Hashable, Accessory: View, InnerAccessory
 
 	public var body: some View {
 		HStack(alignment: .textFieldAlignment, spacing: 0) {
-			VStack(alignment: .leading, spacing: .small3) {
-				HStack(spacing: 0) {
-					if let primaryHeading {
-						VStack(alignment: .leading, spacing: 0) {
-							Text(primaryHeading.text)
-								.textStyle(primaryHeading.isProminent ? .body1HighImportance : .body2Regular)
-								.foregroundColor(primaryHeading.isProminent && isEnabled ? accentColor(isFocused: true) : .app.gray2)
-								.multilineTextAlignment(.leading)
-							if let subHeading {
-								Text(subHeading)
-									.textStyle(.body2Regular)
-									.foregroundColor(.app.gray2)
-									.multilineTextAlignment(.trailing)
+			VStack(alignment: .leading, spacing: .small1) {
+				if primaryHeading != nil || secondaryHeading != nil {
```

Comment:
Is this correct? The whole block will be visible if secondaryHeading is non-nil, but it looks like the block will be empty anyway, unless primaryHeading is also non-nil?

Reply:
Ah, I misread the indentation, I thought that was outside of this `if` block.